lib/bundler/templates/newgem/newgem.gemspec.tt

```diff
@@ -15,6 +13,7 @@ Gem::Specification.new do |spec|
 <%- if config[:mit] -%>
   spec.license       = "MIT"
 <%- end -%>
+  spec.required_ruby_version = Gem::Requirement.new(">= 1.9")
```

Comment:
Any chance this could be configurable? Not sure if the rest of this can be driven from the `.bundle/config`, but that would seem like a good place to have it